api/model/src/main/java/org/projectnessie/model/MergeKeyBehavior.java

```diff
@@ -28,11 +33,61 @@ public interface MergeKeyBehavior {
 
   MergeBehavior getMergeBehavior();
 
+  /**
+   * If present, the current content on the target branch will be compared against this value.
+   *
+   * <p>This parameter is not supported when multiple commits will be generated, which means only
+   * "squashing" merge and transplant operations.
```

Comment:
hmmm, we do not support "unsquashed" merge and "squashed" transplant in API v2, IIRC :thinking: but this param is only for v2. I guess we can say it is supported simply for "merge" operations (also on line 60)?